CHANGES.md

```diff
@@ -1,3 +1,10 @@
+# 4.3.2 (2021-05-25)
+- [NEW] Add new test server env vars
+- [IMPROVED] Make IAM token server configurable in tests
+- [UPGRADED] Use hosted-git-info==2.8.9 dependency.
+- [UPGRADED] Use lodash==4.17.21 dependency.
+- [UPGRADED] Use y18n==4.0.1 dependency.
```

Comment:
I don't think any of these things need to be listed here
1. We usually don't list test changes
2. the package-lock deps only apply if you are installing from the repo (it's not part of the published package, so consumers in other packages have to pin their versions separately anyway - within the range covered by our package.json)

There are more relevant changes incoming from the other PRs.